source/_docs/debug-connections.md

```diff
@@ -19,25 +19,17 @@ Services such as MySQL and SFTP are running; however, the local client is unable
 terminus site wake
 ```
 ### Resolve DNS Issues
-Check to see if you get an IP address returned when you run the following command, replacing `<xxx>` with your site’s UUID:  
+Some ISPs have issues resolving a `drush.in` hostname. Troubleshoot connection errors using the following `dig` command,  replacing `<xxx>` with your [site’s UUID](/docs/sites/#site-uuid):  
 ```
 dig codeserver.dev.<xxx>.drush.in +short
```

Comment:
@peter-pantheon in regards to `codeserver` vs `appserver`, I defer to what CSE recommends. My understanding is that connection attempts from the network's default DNS server will produce similar results across `drush.in` hostnames. Do you have a preference or recommendation on which to use here?